cli/params.go

```diff
@@ -10,18 +10,16 @@ import (
 )
 
 var fetchParamCmd = &cli.Command{
-	Name:  "fetch-params",
-	Usage: "Fetch proving parameters",
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "proving-params",
-			Usage: "download params used creating proofs for given size, i.e. 32GiB",
-		},
-	},
+	Name:      "fetch-params",
+	Usage:     "Fetch proving parameters",
+	ArgsUsage: "[sectorSize]",
 	Action: func(cctx *cli.Context) error {
-		sectorSizeInt, err := units.RAMInBytes(cctx.String("proving-params"))
+		if !cctx.Args().Present() {
+			return xerrors.Errorf("must pass sector size to fetch params for (specify as \"32GiB\", for instance)")
+		}
```

Comment:
This currently breaks functionality, or if this not directly something else did. Passing in a `0` value downloads verifying keys. This "feature" is bad ux in of itself, but we need to at least provide this functionality someplace.

```
$ lotus fetch-params --proving-params 0
```

Reply:
Actually, I guess it doesn't. Carry on!